docs/debugging.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Blazor debugging
+author: danroth27
+description: Learn how to debug Blazor apps.
+manager: wpickett
+ms.author: riande
+ms.custom: mvc
+ms.date: 07/25/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: client-side/blazor/debugging
+---
+# Blazor debugging
+
+[Daniel Roth](https://github.com/danroth27)
+
+[!INCLUDE[](~/includes/blazor-preview-notice.md)]
+
+Blazor has some *very early* support for debugging client-side Blazor apps running on WebAssembly in Chrome. While this initial debugging support is very limited and unpolished, it shows the basic debugging infrastructure coming together.
```

Comment:
I didn't edit these "very" usages out, but they've fallen way out of favor in technical writing. They don't convey any useful information because they're an unfathomable degree of emphasis.

WRT APEX, they don't explicitly prevent the use of the word; however, they do push the "be concise" rule quite a bit. Therefore, they would probably be happy if the word "very" saw limited use.

If you want me to nix them, I'll factor them out. They really aren't *very* :smile: necessary. lol